dashel/dashel-win32.cpp

```diff
@@ -1561,6 +1564,11 @@ namespace Dashel
 						closeStream(strs[r]);
 					}
 				}
+				if(ets[r] == EvClosed)
```

Comment:
missing space between if and (

Reply:
Fixed

Reply:
fixed ( weird convention, it will take some getting used to)